### ISSUE TITLE
[Bug]: Users are not able to access 'Uninstall & Update' buttons which are available on hovering the package

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#11116]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS any

### Worked before?

no

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

See https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1359980/?triage=true

### OS & Environment Details:
Microsoft Windows 10 Enterprise Version 21H1 (OS build: 19043.1052)
Microsoft Visual Studio Enterprise 2022 Preview (64-bit)
Version 17.0.0 Preview 1.0 [31404.170.main]
### Repro Steps:​
Launch latest Visual studio 2022 preview 1
Tab till 'Create new prokect' and press enter
Search for "gRPC Service" project and select it 
Tab till 'Next' button and press enter
Provide proper details and press enter on 'Create' button
Tab till "Solution" and open context menu from keyboard
Tab till 'Add' menu item and press enter
Tab till 'New Project' and press enter
Tab till Search field and search for 'Nunit' project and press enter
Tab till 'Create' button and press enter
Tab till 'Nunit' project in solution explorer and press context menu
Tab till 'Manage NuGet packages' and press enter
Tab till 'Installed' tab item and press enter
Tab till any package and verify user can access 'Uninstall & Update' buttons which are available on hovering the package or not.
### Actual Result:​
Users are not able to access 'Uninstall & Update' buttons which are available on hovering the package
### Expected Result:​
Users should be able to access 'Uninstall & Update' buttons which are available on hovering the package
### User Impact: ​
It would be easy for the users if 'Uninstall & Update' buttons are accessible at the package itself.
### MAS Reference:​
[MAS 2.1.1 Keyboard](https://liquid.microsoft.com/Web/Views/View/410630#Zrex-3A-2F-2Fms-2Eaccessibility-2FRequirements-2F02-2E01-2E01-2F)
